tests/test_workitem_attachments.py

```diff
@@ -30,7 +30,7 @@
     "0ed1417e8e4717524bc91162dcf8633afca686e93f8b036d0bc48d81f0444f56"
 )
 CONTEXT_DIAGRAM_CHECKSUM = (
-    "2b86192f2f65353512e1b4af0e652577d0ca3d0cf8595f5dcfba7d52bcb6d702"
+    "35efc338fccc53e33d29be408bebe2b10f9b43586b9025cb36b4877183d81ae9"
```

Comment:
This is due to a reasonable change in the elk input. In the future there will be tests to test the elk input, see [here](https://github.com/DSD-DBS/capellambse-context-diagrams/issues/204). When this is done, we can drop the check on our side

Reply:
![image](https://github.com/user-attachments/assets/e9de3e99-81f2-4ecc-a1c4-020ffeb757ce)

Old:
![image](https://github.com/user-attachments/assets/2e9e2795-1b1c-4ecf-a590-08982e939604)

New:
![image](https://github.com/user-attachments/assets/287db4b3-af8e-4c39-9395-8d0b9e92cfc8)